app/models/notification.rb

```diff
@@ -68,7 +68,7 @@ def set_notification_recipients
 
   def set_custom_attributes
     return unless notification_blueprint # let validation catch this.
-    self.actions = UINotifications::URLResolver.new(
+    self.actions ||= UINotifications::URLResolver.new(
```

Comment:
I cannot use the URL resolver here, the issue is this notification has no subject. I just want to put an arbitrary link. If I leave this as `=`, my link will get overridden by this (and not work, as there is no subject).